Configurations/10-main.conf

```diff
@@ -222,7 +223,7 @@ my %targets = (
                                               debug   => "-O0 -g",
                                               release => "-O3 -fomit-frame-pointer")),
         cflags           => add(threads("-pthread")),
-        cppflags         => add("-DL_ENDIAN"),
+        lib_cppflags     => add("-DL_DEBIAN"),
```

Comment:
I think I saw you making this typo earlier. There is another occurrence...

Reply:
Maybe you remember the time I wrote `-DL_DEBIAN` ;-)

Reply:
Yes, but even this typos still needs to be corrected :-)

Reply:
Gah!  I didn't see it even when you pointed it out!  Will fix